src/state/search/actions.js

```diff
@@ -140,37 +140,20 @@ export function fetchResults({
 }
 
 /**
- * Ideally the filters should be formatted in the backend, but that seemed to be difficult
- * with elastic search https://github.com/AlexsLemonade/refinebio/pull/974#issuecomment-456533392
+ * Transform filters object that is sent from the API
  */
-function transformElasticSearchFacets(facets) {
+function transformFacets(facets) {
   return {
-    organism: transformFacet(
-      facets['_filter_organism_names']['organism_names']
-    ),
-    technology: transformFacet(facets['_filter_technology']['technology']),
-    publication: transformHasPublicationFacet(
-      facets['_filter_has_publication']['has_publication']
-    ),
-    platform: transformFacet(facets['_filter_platform_names']['platform_names'])
+    organism: facets['organism_names'],
+    // We want to hide `Unknown` from the technologies if it has 0 processed samples
+    technology: facets['technology'], // pickBy(facets['technology'], totalSamples => totalSamples > 0),
```

Comment:
What's the deal with the commented out code here?

Reply:
ups that was me testing. Fixed it